### PR TITLE
add distributed-system

### PR DESCRIPTION
**Pull request by rafaelmnatali**
### Describe your changes
Added `Distribute Systems`

### Related issue number or link (ex: `resolves #issue-number`)
Resolve issue https://github.com/cncf/glossary/issues/821

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/pt-br/distributed-systems.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Sistemas Distribuídas
```
Comment (MrErlison) [inline content/pt-br/distributed-systems.md:2]: typo: "Distribuídas"

File: content/pt-br/distributed-systems.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+tags: ["arquitetura", "", ""]
+---
+
+## O que é
+
+Um sistema distribuído é uma coleção de elementos de computação autônomos conectados por uma rede que aparece para os usuários como um único sistema coerente. Geralmente referidos como [nós](/pt-br/nodes/), esses componentes podem ser dispositivos de hardware (por exemplo, computadores, telefones móveis) ou processos de software. Os nós são programados para alcançar um objetivo comum e, para colaborar, eles trocam mensagens pela rede.
+
+## Problema relacionado     
+
+Inúmeras aplicações modernas hoje são tão grandes que precisariam de supercomputadores para operar. Pense no Gmail ou Netflix. Nenhum computador único é suficientemente poderoso para hospedar toda a aplicação. Ao conectar vários computadores, o poder computacional se torna praticamente ilimitado. Sem a computação distribuída, muitas das aplicações das quais dependemos hoje não seriam possíveis.
```
Comment (MrErlison) [inline content/pt-br/distributed-systems.md:14]: I like "Hoje, inúmeras aplicações modernas são ..." think about it.

File: content/pt-br/distributed-systems.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Tradicionalmente, os sistemas seriam escalados verticalmente. Isso significa adicionar mais CPU ou memória a uma máquina individual. A [escalabilidade vertical](/vertical-scaling/) é demorada, requer tempo de inatividade e atinge seu limite rapidamente.
```
Comment (MrErlison) [inline content/pt-br/distributed-systems.md:16]: Add link to scale (/pt-br/scalability/)
Add /pt-br/ on vertical scaling link

**Comment by MrErlison** [comment]
Hello there, @rafaelmnatali!

Thank you very much for your contribution.  Just a small fix.

File: content/pt-br/distributed-systems.md (hunk 1 above)
Comment (rafaelmnatali) [reply content/pt-br/distributed-systems.md]: updated

File: content/pt-br/distributed-systems.md (hunk 2 above)
Comment (rafaelmnatali) [reply content/pt-br/distributed-systems.md]: updated

File: content/pt-br/distributed-systems.md (hunk 3 above)
Comment (rafaelmnatali) [reply content/pt-br/distributed-systems.md]: updated

**Comment by rafaelmnatali** [comment]
> Hello there, @rafaelmnatali!
> 
> Thank you very much for your contribution. Just a small fix.

thanks for the review @MrErlison. I pushed the fixes as suggested.

**Comment by MrErlison** [comment]
/lgtm

@edsoncelio PTAL

**Review by MrErlison — APPROVED** [review]

**Review by edsoncelio — APPROVED** [review]

**Comment by edsoncelio** [comment]
Thanks @rafaelmnatali!

**Review by brunoguidone — COMMENTED** [review]
LGTM

**Pull request merged**